From b58f21e4bd2a3ca13e9f02416ba0ca53fbdcdc30 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Wed, 21 Feb 2024 08:52:21 -0800
Subject: [PATCH mm-unstable 1/2] Docs/admin-guide/mm/damon/usage: fix a typo on
 a reference link

There was a typo cuasing a document build warning below.  Fix it.

    WARNING: undefined label: 'damon_design_confiurable_operations_set'

Fixes: afc858f0e6db ("Docs/mm/damon: move DAMON operation sets list from the usage to the design document")
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/admin-guide/mm/damon/usage.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/admin-guide/mm/damon/usage.rst b/Documentation/admin-guide/mm/damon/usage.rst
index b33eecfd0e901..952528f0739ba 100644
--- a/Documentation/admin-guide/mm/damon/usage.rst
+++ b/Documentation/admin-guide/mm/damon/usage.rst
@@ -184,7 +184,7 @@ and three directories (``monitoring_attrs``, ``targets``, and ``schemes``)
 exist.
 
 DAMON supports multiple types of :ref:`monitoring operations
-<damon_design_confiurable_operations_set>`, including those for virtual address
+<damon_design_configurable_operations_set>`, including those for virtual address
 space and the physical address space.  You can get the list of available
 monitoring operations set on the currently running kernel by reading
 ``avail_operations`` file.  Based on the kernel configuration, the file will
-- 
2.25.1

